Client/Frontend/Browser/SimpleToast.swift

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+
+
+struct SimpleToastUX {
+    static let ToastHeight = CGFloat(50)
+    static let ToastAnimationDuration = 0.5
+    static let ToastDefaultColor = UIColor(red: 76.0 / 255, green: 158.0 / 255, blue: 255.0 / 255, alpha: 1)
+    static let ToastFont = UIFont(name: "HelveticaNeue-Thin", size: 14)
+    static let ToastDismissAfter = 2.0
+}
+
+struct SimpleToast {
+
+     func showAlertWithText(text: String) {
+        guard let window = UIApplication.sharedApplication().windows.first,
+        let keyboardHeight = KeyboardHelper.defaultHelper.currentState?.intersectionHeightForView(window) else {
+            return
+        }
+        let startHeight = window.frame.height - keyboardHeight
+        let toast = self.createView(window.frame.width, height: CGFloat(startHeight))
+        toast.setTitle(text, forState: .Normal)
+        window.addSubview(toast)
+        animate(toast)
+    }
+
+    private func createView(width: CGFloat, height: CGFloat) -> UIButton {
+        let toast = UIButton(frame: CGRect(x: 0, y: height, width: width, height: SimpleToastUX.ToastHeight))
+        toast.setTitleColor(UIColor.whiteColor(), forState: .Normal)
+        toast.backgroundColor = SimpleToastUX.ToastDefaultColor
+        toast.titleLabel?.font = SimpleToastUX.ToastFont
+        toast.titleLabel?.textAlignment = .Center
+        return toast
+    }
+
+    private func dismiss(toast: UIView) {
```

Comment:
Hm, kind of wish we had a toast class here so we could have a real typed parameter instead of a generic `UIView`.